sdk/keyvault/ci.yml

```diff
@@ -62,7 +62,7 @@ extends:
         - Name: AZURE_KEYVAULT_URL
           Value: "https://non-real-account.vault.azure.net"
         - Name: AZURE_KEYVAULT_HSM_URL
-          Value: "https://non-real-account.managedhsm.azure.net/"
+          Value: "https://non-real-account.vault.azure.net"
```

Comment:
Is it expected that the endpoint moved in 7.5? Otherwise, change looks good.

Reply:
i accidentally changed the endpoint when i regenerated the tests because my KV resources got deleted by tthe automation and i had to recreate them, and the newtestresource is a pain in the behind locally never works out of the box